storage_service/locations/migrations/0007_auto__add_field_file_package__add_field_file_source_package__add_field.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
```

Comment:
This should have a more useful name.  Migrations can be generated with names with ```manage.py schemamigration <app> --auto <name>```; it will prefix the number automatically.